### DISCUSSION TITLE
Wiki Docs too much nesting

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [nvim-neorg/neorg#763] [Ideas]
Would you mind a pull request to remove all this nesting in the docs? It's too cumbersome in the moment to have to open multiple nests to see just a boolean

**Comment by max397574** [comment]
this makes no sense
because the config is just nested

**Comment by LamprosPitsillos** [comment]
https://user-images.githubusercontent.com/61395246/221407789-a692fd19-eed6-491f-812e-64b0b906bdf8.mp4

This is what i was talking about , if i want to see the defaults for concealer for example i have to open all the nests.
I just think it would be easier for it to be something like a code block

```lua
...
,list={
    enabled=true,
    level_1 = {
        enabled = true,
        icon ="•",
        query = "(unordered_list1_prefix) @icon"
    },
},
....
```

**Comment by vhyrro** [comment]
What I would rather have is a separate option to display the whole config as a lua code block. The nesting is pretty crucial and it can't really be removed in any nice way

**Reply by LamprosPitsillos** [reply]
Sounds reasonable , maybe have one more nest `example` per item  or a full config at the top most level ?
I do see why nesting would be crucial for the documentation of many top level headings but is it really crucial for the simple stuff (enabled , item , query etc)?

**Reply by max397574** [reply]
it kinda is
because all this stuff is autogenerated
and everything would be much more complicated if we'd treat some things a different way
